README.md

```diff
@@ -100,14 +101,56 @@ foreach ($yourShipments as $yourShipment) {
         ->setPerson($yourShipment['name'])
         ->setPostalCode($yourShipment['postal_code'])
         ->setFullStreet($yourShipment['full_street']) 
-        ->setCity($yourShipment['city']);
+        ->setCity($yourShipment['city'])
+    );
         
     // Add each consignment to the collection created before
     $consignments
         ->addConsignment($consignment);
 }
 ```
 
+### Create return in the box
+This example creates a consignment and return consignment by adding them to one ```MyParcelCollection()``` and then create and download one PDF with both labels.
+
+```php
+// Create the collection before the loop
+$consignments = new MyParcelCollection();
+
+// Loop through your shipments, adding each to the same MyParcelCollection()
+foreach ($yourShipments as $yourShipment) {
+
+    $consignment = ((ConsignmentFactory::createByCarrierId(PostNLConsignment::CARRIER_ID))
+        ->setApiKey('api_key_from_MyParcel_backoffice')
+        ->setCountry($yourShipment['cc'])
+        ->setPerson($yourShipment['person'])
+        ->setCompany($yourShipment['company'])
+        ->setFullStreet($yourShipment['full_street_input'])
+        ->setPostalCode($yourShipment['postal_code'])
+        ->setCity($yourShipment['city'])
+        ->setLabelDescription($yourShipment['label_description'])
+    );
+        
+    // Add the consignment to the collection and generate teh return consignment
+    $consignments
+        ->addConsignment($consignment)
+        ->generateReturnConsignments(
+            false,
```

Comment:
Misschien hier even een commentje bij zetten, is niet duidelijk vanuit de docs waar die `false` voor is

Reply:
Done